### PR TITLE
GSFLI-134 add docker image build date to logging info

### PR DESCRIPTION
**Pull request by domfre**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -31,7 +31,12 @@ COPY tiles /usr/share/nginx/html/tiles
 COPY --from=builder /pbf /usr/share/nginx/html/tiles/pbf
 COPY template.lua /usr/local/openresty/site/lualib/resty/template.lua
 
-COPY nginx.conf /usr/local/openresty/nginx/conf/nginx.conf
+ARG BUILD_DATE
+ENV image_build_date=[Build_$BUILD_DATE]
+
+COPY nginx.conf /etc/nginx/templates/nginx.conf.template
```
Comment (danduk82) [inline Dockerfile:37]: rename `nginx.conf` to `nginx.conf.template` also in the code, so it is clear that this is indeed a template

File: nginx.conf [hunk 2]
```diff
@@ -18,11 +18,11 @@ http {
     include       mime.types;
     default_type  application/octet-stream;
 
-    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+    log_format  main  '$image_build_date $remote_addr - $remote_user [$time_local] "$request" '
                       '$status $body_bytes_sent "$http_referer" '
                       '"$http_user_agent" "$http_x_forwarded_for"';
 
-    #access_log  /var/log/nginx/access.log  main;
+    access_log logs/access.log main;
```
Comment (danduk82) [inline nginx.conf:25]: does this work when the image is run as non-root? Also are the logs still present in STDOUT of the container?

**Review by danduk82 — CHANGES_REQUESTED** [review]

File: nginx.conf (hunk 2 above)
Comment (danduk82) [reply nginx.conf:25]: otherwise use `/dev/null` as output file if you really need a file

File: Dockerfile (hunk 1 above)
Comment (domfre) [reply Dockerfile]: Changed the name

**Review by danduk82 — APPROVED** [review]

**Pull request merged**